### PR TITLE
Added app sketch

### PR DESCRIPTION
**Pull request by sreejithmunthikodu**
The description will be added. Please let me know any changes in the sketch.

### CODE REVIEW

**Review by fkhan72 — COMMENTED** [review]
where can I see the sketch?

**Comment by sreejithmunthikodu** [comment]
You should be able to see it in the `README.md` file. Also in the `img` folder.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
...
-Movies Dashboard in Python
-Remote change from sree
\ No newline at end of file
+Movies Dashboard in Python  
+
+
+## Description of the app
+
+## Sketch of the app
+!["sketch"](img/sketch.jpg)
```
Comment (fkhan72) [inline README.md]: I think the Genre panel can have check boxes and MPAA Rating panel can have radio buttons to make the interactivity more explicit

Comment (techrah) [reply README.md]: Check boxes for rating is fine too. I might want to look for NC-17 and R films at the same time.

Comment (sreejithmunthikodu) [reply README.md]: But can we select multiple options with radio buttons?

**Comment by techrah** [comment]
> Show tooltip on brush
I'm not exactly sure what "brush" means here. Are you talking about the tooltips when you hover the mouse pointer over the dots?

File: README.md (hunk 1 above)
Comment (sreejithmunthikodu) [reply README.md]: Agree with Ryan

**Comment by sreejithmunthikodu** [comment]
Yes. I thought that is the standard way to write it as shown in the example sketch. I will confirm this.  


> > Show tooltip on brush
> > I'm not exactly sure what "brush" means here. Are you talking about the tooltips when you hover the mouse pointer over the dots?

**Comment by techrah** [comment]
> Yes. I thought that is the standard way to write it as shown in the example sketch. I will confirm this.

I see what you are referring to. In the example sketch, I still don't know what that means. LOL

How about "Show tooltip on hover"?

File: README.md (hunk 1 above)
Comment (techrah) [reply README.md]: @fkhan72 @sreejithmunthikodu Do you agree?

**Comment by sreejithmunthikodu** [comment]
> > Yes. I thought that is the standard way to write it as shown in the example sketch. I will confirm this.
> 
> I see what you are referring to. In the example sketch, I still don't know what that means. LOL
> 
> How about "Show tooltip on hover"?

Sounds good !

File: README.md (hunk 1 above)
Comment (fkhan72) [reply README.md]: sounds good

**Pull request closed without merging**